src/lib/bcmp/bm/bristlemouth.cpp

```diff
@@ -126,6 +82,8 @@ void bcl_init(void) {
   \return pointer to string with ip address
 */
 const char *bcl_get_ip_str(uint8_t idx) {
-  configASSERT(idx < LWIP_IPV6_NUM_ADDRESSES);
-  return (ip6addr_ntoa(netif_ip6_addr(&netif, idx)));
+  (void)idx;
+  //configASSERT(idx < LWIP_IPV6_NUM_ADDRESSES);
+  //return (ip6addr_ntoa(netif_ip6_addr(&netif, idx)));
+  return "none\n";
```

Comment:
Is it possible to also move this over to the IP abstraction? I think it is only used by the CLI command `info` so I don't think it really needs to be a part of bcl.

Reply:
Ahhh yes I meant to do that, thanks for the catch.